DNN Platform/Modules/CoreMessaging/App_LocalResources/View.ascx.resx

```diff
@@ -459,4 +459,10 @@
   <data name="DNNCorp_Wiki_Page.Text" xml:space="preserve">
     <value>Wiki Subscriptions</value>
   </data>
+  <data name="AlreadyFollowingUser.Text" xml:space="preserve">
```

Comment:
@mgatmaytan, are you using these two keys in the code? I cannot find the usage of these new two keys.